docs/using/recipes.md

```diff
@@ -0,0 +1,256 @@
+# Contributed Recipes
```

Comment:
Is there a way we can highlight these in other places with links to these sections? I worry that while they're useful, people looking for a specific solution won't know to look here